MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -171,6 +183,12 @@ struct EventDetails {
         
         modifiedEventDictionary["percentTimeInPortrait"] = percentTimeInPortrait
         modifiedEventDictionary["percentTimeInForeground"] = percentTimeInForeground
+        
+        modifiedEventDictionary["stepIndex"] = stepIndex
+        modifiedEventDictionary["stepCount"] = totalStepCount
```

Comment:
`totalStepCount` is shared here between `modifiedEventDictionary["stepCount"]` and `modifiedEventDictionary["totalStepCount"]`